src/protocols/ExternalRadio/ExternalRadio.h

```diff
@@ -3,12 +3,17 @@
 
 #include "../../TypeDef.h"
 #include "../../Module.h"
+#include "../../ArduinoHal.h"
```

Comment:
This should probably be included only for RADIOLIB_BUILD_ARDUINO.

Reply:
ArduinoHal.h would be empty if RADIOLIB_BUILD_ARDUINO is empty, so is it important to re-check?

Reply:
At the moment, yes, it would be empty, but I would rather avoid this breaking in the future. Also IIRC there's other places where ArduinoHal.h is included which are guarded, so let's make it consistent. It's a compile-time check anyway.